webtau-cli/src/main/java/com/twosigma/webtau/cli/CliValidationResult.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twosigma.webtau.cli;
+
+import com.twosigma.webtau.cli.expectation.CliOutput;
+import com.twosigma.webtau.reporter.TestStepPayload;
+
+import java.util.*;
+
+public class CliValidationResult implements TestStepPayload {
```

Comment:
Kotlin data classes would be nice here 😁

Reply:
can't argue with that.

Reply:
We could in theory use them, Kotlin happily coexists with Java (and presumably Groovy too).

Reply:
@tsiq-karold I learnt the hard way not to build a core login in non Java lang.